include/linux/soundwire/sdw.h

```diff
@@ -812,6 +813,7 @@ struct sdw_master_ops {
  */
 struct sdw_bus {
 	struct device *dev;
+	int id;
```

Comment:
make sure the member and kernel doc are in the same order.
This will report errors with make W=1

Reply:
Thanks, it was a mistake of rebasing. :)